Azure/azure-cli
# {Find} az find: Send User ID hash for rate limiting

**Description**  
In order to better handle the increased traffic, the Aladdin service is adding hashed User IDs to the request header in order to accurately rate limit clients. These should not interfere with normal usage, just prevent bad actors (most likely run-a-way scripts) from slowing down or breaking the service for other users.

The hashed ID is only used for the rate limiting and is only stored in memory for a temporary amount of time. It is not stored in telemetry.

**Testing Guide**  
No change in command usage.

This checklist is used to make sure that common guidelines for a pull request are followed.
